cookiecutter_context.json

```diff
@@ -0,0 +1,60 @@
+{
+    "formal_name": [
+        "First, we need a formal name for your application. ",
+        "This is the name that will be displayed to humans whenever the ",
+        "name of the application is displayed. It can have spaces and ",
+        "punctuation if you like, and any capitalization will be ",
+        "used as you type it. "
+    ],
+    "app_name": [
+        "Next, we need a name that can serve as a machine-readable ",
+        "Python package name for your application. ",
+        "\n",
+        "This name must be PEP508-compliant. This means the name: ",
+        " * may only contain letters, numbers, hyphens and underscores; ",
+        " * can't contain spaces or punctuation; and ",
+        " * can't start with a hyphen or underscore. "
+    ],
+    "project_name": [
+        "Briefcase can manage projects that contain multiple applications, ",
+        "so we need a Project name. ",
+        "If you're only planning to have one application in this project, ",
+        "you can use the formal name as the project name."
+    ],
+    "description": [
+        "Now, we need a one line description for your application. "
+    ],
+    "author": [
+        "Who do you want to be credited as the author of this application? ",
+        "This could be your own name, or the name of the company you work for."
+    ],
+    "author_email": [
+        "What email address should people use to contact the developers ",
+        "of this application? This might be your own email address, ",
+        "or a generic contact address you set up specifically for this ",
+        "application."
+    ],
+    "bundle": [
+        "Now we need a bundle identifier for your application. ",
+        "App stores need to protect against having multiple applications ",
+        "with the same name; the bundle identifier is the namespace they use ",
+        "to identify applications that come from you. ",
+        "The bundle identifier is usually the domain name of your company or ",
+        "project, in reverse order.\n",
```

Comment:
I think we should pick a style and stick with it - either use standalone `""` or `\n` for blank lines, but not both. Personally, I lean to "", as it renders most like the final output, but I could be convinced otherwise.

Reply:
"" will not render as a line break, whereas `\n` will. I think your suggestion here is that I should put the `\n` on a separate line by itself, rather than trailing the text here?

Reply:
Ok - Looks like I misunderstood how this text is being rendered.

I'm suggesting that the text in the template should be rendered with 66-char string lengths, and then be joined with `"\n"`; that way, the template shows you *exactly* what is going to be rendered to the end user, including line breaks. That also means that `""` on a line by itself appears like a paragraph break in text, *and* will be displayed as such to the end user. 

Joining with `''` means these lines are going to render as exceedingly long paragraphs, which isn't great for readability.